api/src/db/DynamoUserDataClient.ts

```diff
@@ -141,6 +141,31 @@ export const DynamoUserDataClient = (
     return await search(statement);
   };
 
+  const getUsersWithBusinesses = async (
+    ExclusiveStartKey?: any
```

Comment:
This value can be undefined, we should hace tests, will results correctly come back from the db?